src/Tool/Lerna.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Silverorange\PackageRelease\Tool;
+
+use Symfony\Component\Console\Output\OutputInterface;
+use Silverorange\PackageRelease\Console\ProcessRunner;
+
+/**
+ * @package   PackageRelease
+ * @author    Meg Mitchell <meg@silverorange.com>
+ * @copyright 2020 silverorange
+ * @license   http://www.opensource.org/licenses/mit-license.html MIT License
+ */
+class Lerna
+{
+    public static function verify(OutputInterface $output): bool
+    {
+        $command = 'yarn global add lerna';
+
+        return (new ProcessRunner(
+            $output,
+            $command,
+            'verifying Lerna installation',
+            'verified Lerna installation',
+            'failed to verify Lerna installation'
+        ))->run();
+    }
+
+    public static function bootstrap(OutputInterface $output, Array $scopes): bool
+    {
+        $command = 'yarn lerna bootstrap';
```

Comment:
this should be `lerna bootstrap`

Reply:
`lerna bootstrap` didn't work during testing since it wasn't globally installed, but since it is now I will update the PR 👍